src/component/container/element-list.tsx

```diff
@@ -128,38 +136,76 @@ export class ElementList extends React.Component {
 				store.execute(ElementLocationCommand.addChild(element, draggedElement));
 				store.setSelectedElement(draggedElement);
 			},
-			children: items,
-			active: element === selectedElement
+			children: [...slots, ...defaultSlotItems],
+			active: element === selectedElement && !store.getSelectedSlot()
```

Comment:
i have to check this. Is this just a check if a selected slot is selected and not a element?